src/main/java/github/scarsz/discordsrv/api/commands/SlashCommand.java

```diff
@@ -0,0 +1,39 @@
+package github.scarsz.discordsrv.api.commands;
+
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import net.dv8tion.jda.api.interactions.InteractionHook;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Annotation to denote the attached method as a slash command handler.
+ * <strong>Method must have exactly one parameter of type {@link SlashCommandEvent}.</strong>
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.METHOD)
+public @interface SlashCommand {
+
+    /**
+     * The command path that shall be handled by this method
+     * @return the command path
+     * @see SlashCommandEvent#getCommandPath()
+     */
+    String path();
```

Comment:
Example here would be nice, explaining what parts of the command (command + subcommand group + subcommand) are included and how they need to be formatted (separated by slashes)